src/js/containers/review/ReviewDataContainer.jsx

```diff
@@ -56,6 +56,10 @@ class ReviewDataContainer extends React.Component {
             })
             .then((name) => {
                 submission.agency_name = name;
+                return ReviewHelper.fetchSubmissionComments(this.props.params.submissionID);
+            })
+            .then((comments) => {
+                submission.file_comments = comments;
```

Comment:
Does it make sense to async all these things in parallel? The main data, obligations, and narratives don't depend on each other.